src/app/dataset/workflow/workflow.component.html

```diff
@@ -430,6 +429,60 @@
           >{{ 'LINK_CHECKING' | renameWorkflow }}
           <input type="checkbox" formControlName="pluginLINK_CHECKING" id="pluginLINK_CHECKING"
         /></label>
+        <div
+          class="form-group"
+          *ngIf="
+            !workflowForm.get('pluginLINK_CHECKING')!.disabled &&
+            workflowForm.get('pluginLINK_CHECKING')!.value
+          "
+        >
+          <div class="form-group-radios">
+            <div>
```

Comment:
Is this div necessary?

Reply:
This was copied and pasted from the harvest url radios.  I've removed the redundant div elements from both places.